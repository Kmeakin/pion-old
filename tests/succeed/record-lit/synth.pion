//~ run = "${PION} elab ${FILE}"

{x = false, y = 1, z = 2}
