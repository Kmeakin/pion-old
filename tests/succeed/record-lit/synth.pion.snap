exit status: 0
stdout:
{ x = false, y = 1, z = 2 } : { x : Bool, y : Int, z : Int }

stderr:
